pages/story/[id].tsx

```diff
@@ -95,7 +114,25 @@ export async function getStaticPaths() {
   }
 }
 
-export async function getStaticProps({ params }) {
-  const story = await get(params.id)
-  return { props: { story } }
+export async function getStaticProps({ params }): Promise<GetStaticPropsResult<Props>> {
+  try {
+    /**
+     * Needed to use `as` keyword, but this should be refactored.
+     */
+    const story = (await get(params.id)) as Story
+    console.log('it got here')
```

Comment:
Remove leftover log? 😃

Reply:
Haha thanks, I hadn't noticed it! Removed now.